pkg/autodetect/main.go

```diff
@@ -109,8 +111,29 @@ func (b *Background) autoDetectCapabilities() {
 	b.cleanDeployments(ctx)
 }
 
+func (b *Background) isInListenedGroups(group *metav1.APIGroup) bool {
+	for _, groupName := range groupsListened {
```

Comment:
`map` might be more efficient here. The code checks if a given group should be handled or not.